buildSrc/src/main/kotlin/dependencies/Libraries.kt

```diff
@@ -0,0 +1,30 @@
+package dependencies
+
+object Libraries {
+    val GWTEXPORTER = "org.timepedia.exporter:gwtexporter:${Versions.GWTEXPORTER}"
```

Comment:
Is the format of this Libraries.kt a standard Gradle thing? If not, do these constants get used anywhere?

Reply:
It is not *the* standard Gradle thing, because there is no common convention. It is however one of the formats commonly used in the real world to define a set of fixed version dependencies.

The constants get used all over the place in different `build.gradle.kts` scripts throughout the entire project.